SeleniumWebDriverBasics/SeleniumWebDriverBasics/Tests/BaseTest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using OpenQA.Selenium;
+using SeleniumWebDriverBasics.WebDriver;
+using SeleniumWebDriverBasics.WebObjects;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SeleniumWebDriverBasics.Tests
+{
+    public abstract class BaseTest
+    {
+        private readonly HomePage homePage = new HomePage();
+        private readonly LoginPage loginPage = new LoginPage();
+        private readonly EmailPage emailPage = new EmailPage();
+        protected string login = "katetest.selenium@yandex.com";
+        protected string password = "testSelenium001";
```

Comment:
Login and password should not be present in the BaseTest. Please remove them and use values from the App.config.